packages/app-mobile/components/screens/status.tsx

```diff
@@ -127,6 +153,7 @@ class StatusScreenComponent extends BaseScreenComponent {
 									<Text style={style}>{item.text}</Text>
 									{retryAllButton}
 									{retryButton}
+									{ignoreButton}
```

Comment:
The "ignore" and "retry" buttons should be swapped. Currently, it's easier to click "ignore" than "retry" ("ignore" is closer to the edge of the screen). It should probably be the other way around:

![screenshot: ignore to the right of retry, arrows suggest switching them](https://github.com/laurent22/joplin/assets/46334387/6d0eff19-0d60-4d1a-92e8-e70c1d5278b7)

Reply:
Changed in 81f32710624b02599bfab72d8b01e6283720d742